### PR TITLE
LUI-167: Should be able to find a program workflow matching a given id

### PR DESCRIPTION
**Pull request by rubailly**

### CODE REVIEW

**Comment by dkayiwa** [comment]
@rubailly did you get a chance to look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by sherrif10** [comment]
Thanks @rubailly  did you see travis failure

File: pom.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@
 	</modules>
 	
 	<properties>
-		<openmrsPlatformVersion>2.0.0</openmrsPlatformVersion>
+		<openmrsPlatformVersion>2.2.0</openmrsPlatformVersion>
```
Comment (sherrif10) [inline pom.xml:54]: @rubailly  Any reason why you are not updating it to platform_version 2.3.2

Comment (HerbertYiga) [reply pom.xml:54]: hi @rubailly ,did you try building this locally before pushing the changes ?

**Comment by rubailly** [comment]
@dkayiwa -sure, I'll add a ticket and update the PR.
@sherrif10 the method I wanted to get access to is available from 2.2.0 upward, so I jumped right to 2.2.0 
I might have to change this thought (...@HerbertYiga ), might be the reason tests are failing 

I'll update this PR and ping you.

**Pull request closed without merging**